lib/ide/parser/src/lib.rs

```diff
@@ -9,6 +9,8 @@ use crate::prelude::*;
 
 use std::panic;
 use std::ops::DerefMut;
+use crate::api::IDMap;
```

Comment:
"use crate::" declarations should be places between prelude and other use declarations.